web-service/templates/base.html

```diff
@@ -1,20 +1,58 @@
+{% macro player_table(players) -%}
+<table>
+  <tr>
+    <th>Player Name</th>
+    <th>Skating Rating</th>
+    <th>Shooting Rating</th>
+    <th>Checking Rating</th>
+  </tr>
+  {% for player in players %}
+  <tr>
+    <td>{{ " ".join([player.first_name, player.last_name]) }}</td>
+    <td>{{ player.skating }}</td>
+    <td>{{ player.shooting }}</td>
+    <td>{{ player.checking }}</td>
+  </tr>
+  {% endfor %}
+</table>
+{%- endmacro %}
+
 <!DOCTYPE html>
 <html lang="en">
   <head>
     <title>Squad Matchmaker</title>
     <link href="{{ url_for('static', filename='style.css') }}" rel="stylesheet">
   </head>
   <body>
+    <style>
+      table, th, td {
```

Comment:
You have a css document, why put this stuff here instead of applying it to a class or something there?

Reply:
Legit tried and it refused to work. I don't understand why doing it here worked and doing the same stuff in the stylesheet didn't, so I cut my losses with it and moved on to other parts of the problem.